evap/static/ts/src/quick-review-slider.ts

```diff
@@ -447,7 +451,9 @@ export class QuickReviewSlider {
             // because the element has zero opacity until we add `.active`.
             nextActiveElement.classList.remove("to-left", "to-right");
             nextActiveElement.classList.add(`to-${direction}`);
-            requestAnimationFrame(() => nextActiveElement.classList.add("active"));
+            requestAnimationFrame(() => {
+                nextActiveElement.classList.add("active");
+            });
```

Comment:
Personally, I'd be fine with allowing single statements without an explicit block, but I don't care too much.

Reply:
we can allow the "ArrowShorthand", I think this makes definitely sense :+1:

Reply:
I've dropped the commit in question and further enabled a rule that forces using the arrow shorthand "as-needed" (enforces no braces where they can be omitted). Might be too restrictive though :man_shrugging: